src/fields/fieldp128/mod.rs

```diff
@@ -124,7 +122,26 @@ impl LagrangePolynomialFieldElement for FieldP128 {
     }
 
     fn mul_inv(&self) -> Self {
-        mul_inv_field_order(self, BigUint::from_bytes_le(Self::MODULUS_BYTES.as_slice()))
+        // The multiplicative group of any finite field is a group with order one less than the field
```

Comment:
It's unfortunate to duplicate this long comment in each impl. Maybe it could be hoisted up to the trait's definition of `mul_inv`?